pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -32,13 +33,23 @@ const (
 	otelExporterSmpEndpointDefaultValue    = "http://cloudwatch-agent.amazon-cloudwatch:4315"
```

Comment:
If we use `OTEL_EXPORTER_OTLP_METRICS_PROTOCOL:  http/protobuf`, do we need metrics endpoint to be something like: `http://cloudwatch-agent.amazon-cloudwatch:4316/v1/metrics`?

Reply:
We only care about the `OTEL_AWS_SMP_EXPORTER_ENDPOINT` for appsignals metrics. If the endpoint says port `4315` (which it will for EKS) then the SMP exporter will be a grpc exporter.